src/main/java/com/cloudbees/jenkins/support/impl/GCLogs.java

```diff
@@ -83,7 +83,7 @@ public void addContents(@NonNull Container result) {
                         "but file '" + gcLogFileLocation + "' not found");
                 return;
             }
-            result.add(new FileContent(GCLOGS_BUNDLE_ROOT + "gc.log", file));
+            result.add(new UnFilteredFileContent(GCLOGS_BUNDLE_ROOT + "gc.log", file));
```

Comment:
Without filtering, not private info here, avoid destroying the file and make it parseable by tools.